docs/v1/Makefile

```diff
@@ -29,4 +34,4 @@ images:
 build_spec_with_images: images all
 
 clean:
-	/bin/rm -f ${SPEC}.pdf ${SPEC}.html
\ No newline at end of file
+	/bin/rm -rf build resources/figs/*.png
```

Comment:
If we delete the `resources/figs/*.png` files, may as well also delete the `*.svg` files in that directory, too, yes?

Reply:
@Dscano @jafingerhut Is this resolved? I would like to merge this so incoming PRs can be synced.